trigger.tex

```diff
@@ -36,6 +36,34 @@ \chapter{Trigger Module}
     \end{enumerate}
 \end{commentary}
 
+\section{Native M-Mode Triggers}
+\label{sec:mmtrigger}
+
+Triggers can be used for native debugging. On a fully featured system triggers
+will be set using \Fu or \Fs, and when firing they can cause a debug exception
```

Comment:
There is no such thing as “debug exception”. To avoid confusion we should use the same term as the priv spec, which I think is breakpoint exception.

Reply:
Yes. Thanks for catching that.